doc/tools/docgram/common.edit_mlg

```diff
@@ -383,15 +383,24 @@ evar_instance: [
 | OPTINREF
 ]
 
+(* OPT [ "|" binders ] is not supported for Record *)
+record_definition: [
+| opt_coercion ident_decl binders OPT [ ":" type ] OPT [ identref ] "{" record_fields "}"  decl_notations
+]
+
+variant_definition: [
+| opt_coercion ident_decl binders OPT [ "|" binders ] OPT [ ":" type ] OPT "|" LIST1 constructor SEP "|" decl_notations
```

Comment:
```suggestion
| opt_coercion ident_decl binders OPT [ "|" binders ] OPT [ ":" type ] := OPT "|" LIST1 constructor SEP "|" decl_notations
```

Reply:
Got it.  The correct line is a little different from your suggestion.  (Would be nice if doc_grammar could do more checking of the edits had clearer editing instructions for some of these cases.  But that's a project.)

`| ident_decl binders OPT [ "|" binders ] OPT [ ":" type ] OPT ( ":=" OPT "|" LIST1 constructor SEP "|" ) decl_notations
`

Reply:
Actually I did not put the part with `:=` optional on purpose: cf. https://github.com/coq/coq/pull/11868#issuecomment-602058738